src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/controller/PersonController.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.personrecord.controller
+
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.personrecord.model.PersonDTO
+import uk.gov.justice.digital.hmpps.personrecord.service.PersonRecordService
+
+@RestController
+class PersonController(
+  private val personRecordService: PersonRecordService,
+) {
+
+  companion object {
+    private val log = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @GetMapping("/person/{person-id}")
```

Comment:
worth validating the UUID here with a regex or use java.util.UUID instead of String or other means?